frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/create-sc/wizard-pages/donut-chart.tsx

```diff
@@ -32,6 +40,16 @@ export const DiscoveryDonutChart: React.FC<DiscoveryDonutChartProps> = ({ state,
   const nodes = getNodes(state.showNodesListOnLVS, state.nodeNamesForLVS, state.nodeNames);
   const { podStatusOuterRadius: radius } = calculateRadius(220);
 
+  React.useEffect(() => {
+    if (discoveriesLoaded && !discoveriesLoadError && discoveriesData.length) {
+      if (discoveriesData[0].status?.conditions[0]?.type === 'Available') {
```

Comment:
We dont want to do any additional fetch for local volume discovery, rather watch for local volume discovery results.
The PR #8256 adds that and we shall leverage that.
Once all discoveries get created i.e equivalent to the nodes we chosen, then we can say data is loaded and hence stop loading. Other  conditions like loadError etc, needs to be taken care as well.
https://github.com/openshift/console/blob/4cfef73693dc256b6e6a1c5d89f1262df03027aa/frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/create-sc/wizard-pages/selected-capacity.tsx#L89

Reply:
@afreen23 Okay! I am working on it.